src/EnvatoConfig.php

```diff
@@ -63,4 +69,19 @@ static function ($name, $data) {
             $this->config['packages']
         );
     }
+
+    protected function loadDotenv(): void
+    {
+        $cwd = \getcwd();
+        if ($cwd && \file_exists($cwd . DIRECTORY_SEPARATOR . '.env')) {
```

Comment:
Please use boolean conditions.

```suggestion
        if (\is_string($cwd) && \file_exists($cwd . DIRECTORY_SEPARATOR . '.env')) {
```